### DISCUSSION TITLE
how could I try all the models?

### DISCUSSION THREAD

**Discussion by zzzgit** [coqui-ai/TTS#2512] [General Q&A]
Hi, I want to try all the models and vocoders to determine the best for me. However, installing all of them will take lots of disc space, I think. Are there any online resources that I can use?

**Comment by thorstenMueller** [comment]
Maybe this helps you. My Coqui TTS model audio sample comparison videos.

* https://youtu.be/HojuVmW5LUI
* https://youtu.be/Vnjv2L31eyQ
